### DISCUSSION TITLE
Unexpected behavior using top-01.

### DISCUSSION THREAD

**Discussion by DamonGuzman** [tailwindlabs/tailwindcss#12427] [Help]
**What version of Tailwind CSS are you using?**
v3.3.5

**What build tool (or framework if it abstracts the build tool) are you using?**
Next.js 14.0.2

**What version of Node.js are you using?**
v20.5.1

**What browser are you using?**
Microsoft Edge & Chrome

**What operating system are you using?**
Windows 11

https://github.com/DamonGuzman/tailwind-bug-public.git

**Describe your issue**
Adding top-01 is behaving unexpectedly. Instead of positioning it 1px  from top or 0px at all because it is an invalid class, it is adding what seems to be very close to 2.5rem of spacing from the top. 
I experienced similar effects when using top-02, top-03, etc..

**Comment by adamwathan** [comment]
Hey! I can't reproduce this, as you mentioned `top-01` isn't a class in Tailwind and does nothing:

https://play.tailwindcss.com/pImyh5pciA

**Reply by DamonGuzman** [reply]
Did you try cloning the repo I created for it? It has a minimal reproducible example that shows 3 different instances of the effect.

**Reply by wongjn** [reply]
The repository link gives 404 – perhaps it is private?

**Reply by wongjn** [reply]
> My apologies, it was set to private like the rest of my repositories. I've made it public. Let me know if you're able to reproduce using the repo.[^1]

As adamwathan says, `top-01` does nothing as evidenced by [this Tailwind Play](https://play.tailwindcss.com/J2qqzfwZ5t). You can observe that nothing changes if you remove the `top-01` class. If you'd like the absolutely-positioned element to be at `1px` or `0px` from the top of its position parent, consider applying `top: 1px` or `top: 0px` via `top-px` or `top-0` respectively: https://play.tailwindcss.com/gH37Aoupzh

[^1]: [Comment](https://github.com/tailwindlabs/tailwindcss/discussions/12427#discussioncomment-8042226)

**Comment by DamonGuzman** [comment]
My apologies, it was set to private like the rest of my repositories. I've made it public. Let me know if you're able to reproduce using the repo.

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/12427#discussioncomment-8044795 to keep the reply chain together.
